audit/mapping.go

```diff
@@ -0,0 +1,51 @@
+package admin
+
+import (
+	"gorm.io/gorm"
+)
+
+// RequestToEntity request to entity
+func RequestToEntity(req *Request) (entity Entity) {
+	if req != nil {
+		entity.LastUpdatedBy = req.RequestedBy
+		entity.UpdateReason = req.Reason
+	}
+	return
+}
+
+// EntityToModel base entity to base model
```

Comment:
Shall we explain more abt the `m` parameter?
If I have to use this function, I don't know what I should pass for `m`.
Same for `ActionEntityToAction` below.